framework/test_app/runners/k8s/k8s_base_runner.py

```diff
@@ -505,114 +505,159 @@ def _create_deployment(self, template, **kwargs) -> k8s.V1Deployment:
         )
         return deployment
 
-    def _create_gamma_route(self, template, **kwargs) -> k8s.GammaHttpRoute:
+    def _create_gamma_route(
+        self,
+        template: str,
```

Comment:
Can we change template to kwargs?

Reply:
All kwargs become template vars. That's why template is expected to be a positional arg. This pattern is all across `_create_*` methods in this class.